android/app/src/main/res/values/styles.xml

```diff
@@ -7,11 +7,15 @@
         <item name="android:colorEdgeEffect">@color/gray4</item>
         <item name="colorAccent">@color/accent</item>
         <item name="android:statusBarColor">@color/white</item>
-        <item name="android:windowLightStatusBar">true</item>
+        <item name="android:windowLightStatusBar" tools:ignore="NewApi">true</item>
```

Comment:
`android:windowLightStatusBar` and `android:windowLightNavigationBar` are not available on Android 21 (the min version), `tools:ignore` silents the Android studio linter.